src/plugins/reporter-ureport.c

```diff
@@ -90,12 +95,23 @@ int main(int argc, char **argv)
                           _("attach short text (requires -a|-A, conflicts with -D)")),
         OPT_BOOL('O', "comment-file", &comment_file,
                           _("attach short text from comment (requires -a|-A, conflicts with -d)")),
+
+        /* va l ue */
+        OPT_STRING('l', "value", &attach_value, "DATA",
+                          _("attach value (requires -a|-A and -t, conflicts with -V)")),
+        OPT_STRING('L', "value-rt", &attach_value_from_rt, "FIELD",
+                          _("attach data of FIELD [URL] of the last report result (requires -a|-A and -r|-R and -t, conflicts with -v)")),
+
+        OPT_STRING('r', "report-result-type", &report_result_type, "REPORT_RESULT_TYPE",
+                          _("use REPORT_RESULT_TYPE when looking for FIELD in reported_to (used only with -V)")),
+        OPT_STRING('T', "type", &attach_type, "TYPE",
+                          _("attach DATA as ureporte attachment TYPE (used only with -v|-V)")),
         OPT_END(),
     };
 
     const char *program_usage_string = _(
         "& [-v] [-c FILE] [-u URL] [-k] [-t SOURCE] [-h CREDENTIALS]\n"
-        "  [-A -a bthash -B -b bug-id -E -e email -O -o comment] [-d DIR]\n"
+        "  [-A -a bthash -B -b bug-id -E -e email -O -o comment -R -r url] [-d DIR]\n"
```

Comment:
Eek, this is a left-over from the very first version of this commit. Good catch. Thanks!